discord_bots/models.py

```diff
@@ -488,6 +501,28 @@ class SkipMapVote:
     )
 
 
+@mapper_registry.mapped
+@dataclass
+class PersistentView:
+    """
+    Stores persistent view information.
+    To be used when re-initializing views in cogs.
+    """
+
+    __sa_dataclass_metadata_key__ = "sa"
+    __tablename__ = "persistent_view"
+
+    view_id: str = field(
+        init=False,
+        default_factory=lambda: str(uuid4()),
+        metadata={"sa": Column(String, primary_key=True)},
+    )
+    message_id: int = field(metadata={"sa": Column(BigInteger, nullable=False)})
+    view_type: str = field(
+        metadata={"sa": Column(String, nullable=False)},
+    )
```

Comment:
```suggestion
```
this doesn't seem necessary - in progress game knows what to do with the view because it has a direct relationship via its column

Reply:
That's fair, the intention was to give something at a glance that would show in persistent_view what that view is referring to.
I don't believe we'd ever be calling that table directly though, it's more a case of calling the IPG & then joining based on it's column back to the view to get the message_id (and in future, guild & channel)